M2/Macaulay2/m2/mathml.m2

```diff
@@ -59,7 +59,29 @@ mathML FunctionApplication := m -> (
      then concatenate (mfun, bigParenthesize margs)
      else concatenate (bigParenthesize mfun, bigParenthesize margs)
      )
-mathML MatrixExpression := x -> concatenate( "<mrow><mo>(</mo>", mtableML x, "<mo>)</mo></mrow>", newline )
+
+-- zero-width "phantom" element to make sure that degrees line up
+mphantom := x -> concatenate(
```

Comment:
We don't need to do this now, but fixing the spacing between terms (currently the + or - spacing is less than inside a monomial.  Also, sometimes the dot (or previously, asterisk) is included, sometimes not).  See conversation for an example...

Reply:
I think the big problem with how we represent these `Product` objects (not just by `mathML`, but in `net(Product)` too) is not knowing what the factors are.  We could totally represent `Product{x, y}` by `xy`, but we want `Product{2,3}` to be `2 * 3` and not `23`.